cmd/bootstrap/main.go

```diff
@@ -86,12 +88,21 @@ func main() {
 	// Create new service
 	svc := newService(auth, db, logger, esClient, cfg)
 
+	//create an instanceID for the service
```

Comment:
No need for this comment.

Reply:
This is a part of the configuration (i.e. setup) so move it closer to config loading at the beginning and exit (log.Fatalf) if it's impossible to get an instance ID.